O/Opus/build_tarballs.jl

```diff
@@ -3,16 +3,14 @@
 using BinaryBuilder
 
 name = "Opus"
-version = v"1.3.1"
+version = v"1.5.2"
 
 # Collection of sources required to build Opus
 sources = [
-    ArchiveSource("https://archive.mozilla.org/pub/opus/opus-$(version).tar.gz",
-                  "65b58e1e25b2a114157014736a3d9dfeaad8d41be1c8179866f144a2fb44ff9d"),
+    ArchiveSource("https://gitlab.xiph.org/xiph/opus/-/archive/v$(version)/opus-v$(version).tar.bz2",
```

Comment:
Is this an auto generated tarball? If so, it has the same problem as on github (but we don't have the check for gitlab just because I don't know what's the regex)

Reply:
Probably. Switching to `GitSource`.